docs/en-us/cloneset.md

```diff
@@ -345,7 +345,114 @@ spec:
     paused: true
 ```
 
-### PreUpdate and PostUpdate
+### Lifecycle hook
 
-`PreUpdate` and `PostUpdate` can allow users to specify extra tasks before and after Pod update.
-This feature will be available in future release.
+Since v0.6.1, kruise has supported lifecycle hook for CloneSet.
+
+Each Pod managed by CloneSet has a clear state defined in `lifecycle.apps.kruise.io/state` label:
+
+- Normal
+- PreparingUpdate
+- Updating
+- Updated
+- PreparingDelete
+
+Lifecycle hook allows users to do something during Pod deleting and before/after in-place update.
```

Comment:
replace do something  with some real use cases， for example, remove pod from service endpoints ?